worlds/generic/docs/advanced_settings_en.md

```diff
@@ -140,6 +141,31 @@ Options taking a choice of a number can also use a variety of `random` options t
 * `random-range-low-#-#`, `random-range-middle-#-#`, and `random-range-high-#-#` will choose a number at random from the
   specified numbers, but with the specified weights
 
+### Custom Item Pool
+
+`custom_item_pool` allows you to have complete control over the items generated for your game. Sub-options are:
+
+* `modify`: Allows you to increase or decrease the number of any given items by a given amount. Example: `Power Star: 2`
```

Comment:
The purpose of having the example yamls is to consolidate all examples and organization. By mixing examples here it adds clutter when you could be more descriptive instead.